features/users/login.feature

```diff
@@ -15,19 +15,3 @@ Feature: Login
     Given "hello@world.com" is a user
     When I login as "hello@world.com" with an incorrect password
     Then I should not be logged in
-
-  Scenario: Redirected to original page on login
-    Given I am a logged out user
-    And I am a member of a hidden group
-    When I visit the group page
-    And I log in
-    Then I should see the group page
-
-  @javascript
-  Scenario: Redirected to previous public page on login
```

Comment:
There's a couple of things here which are actually lack of feature parity; redirecting you to the right place after login and unsubscribing via email link being the two I remember off the top of my head.